### PR TITLE
Start pages

### PR DESCRIPTION
**Pull request by FravBox**
(I'm Vars, btw)

- Using https://github.com/pages-themes/midnight/tree/master?tab=readme-ov-file template
- Create index page & base site style similar to https://bentovid.com/vpr 
- Add all risk factors as their own md files. The title of each file is the first word of the risk heading (so "flashing and flickering" was just "flashing" )

I copy/pasted the risks from the site into a markdown format.
I did not copy over any of the bentovid.com gif links (the ones used for examples).    
Mostly because I was lazy and didn't want to add all the images to the github. But I can always do it later if you want them.

I put `TODO` in some files (can't recall how many).

I wasn't sure how you wanted to organize the site/files, so I guessed. I really just made this to get us started. You can change stuff around of course.     
I also am not familiar with how jekyll works, so I don't know how to do much of anything dynamic.     
(I spent a whole hour just trying to get the github image to appear on all pages :skull: )

I'll probably add the other info later. But it may need to be rewritten/updated.

Sorry about my commit messages.

### CODE REVIEW

**Review by SerTalithaT — APPROVED** [review]
Small things I'd fix but not necessary for this starting point. I'll circle back myself.
Thanks for doing this!! 😄

**Comment by FravBox** [comment]
What things would you change?
I can do it if I work on this before you

**Comment by SerTalithaT** [comment]
@FravBox I went ahead and merged but I'll add comments for reference

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
+<img title="Enby, a pink octopus with sunglasses, by artist James Eden" src="img/enby_octopus_by_jameseden.png" height="500px">
+
+ Vidding is the name for the hobby of editing fan-videos. Photosensitive individuals are often overlooked when creating and sharing content, making it hard for them to find videos that are safe to view. The VPR system seeks to change this by offering information on how to relay content advisories for common photosensitive risks.
+
+If you'd like to read more about what can cause seizures to further help rate your video, please see [WCAG](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html) (Web content accessibility guidelines) and/or [Prodicle's Photosensitivity Prevention Best Practice Guideline - Production](https://web.archive.org/web/20211130113204/https://help.prodicle.com/hc/en-us/articles/360044218174-Photosensitivity-Prevention-Best-Practice-Guideline-Production) (TODO: find a more reliable url or copy the content elsewhere as it's no longer public.)
+
+This page is designed as a community-based collaboration. You can submit issues and pull requests to [the GitHub repository here]({{ site.github.repository_url }}).
```
Comment (SerTalithaT) [inline README.md:8]: Link doesn't work on README - consider restructuring repo file tree so there is a separate section for just repo stuff vs all the page details?
Top-level repo README could then have easy link to the page

File: risks/readme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Common Photosensitive Risk Categories
+
+Remember: Your source can have these naturally! Your video can be a risk without adding any effects!
+
+TODO: Do we need to explain that "risk categories" are "triggers" ?
```
Comment (SerTalithaT) [inline risks/readme.md:5]: Have theming option so to-dos show up in different colors on the page?

File: risks/blurs.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+# Blurs (**B**)
```
Comment (SerTalithaT) [inline risks/blurs.md:2]: Figure out how to get the risks to actually show up

File: README.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+
+<div class="notice">TL;DR: It's like a CW, but for photosensitivities</div>
```
Comment (SerTalithaT) [inline README.md:10]: Add a quick summary of the risks with more obvious redirection to the "more info"

**Pull request merged**